common/persistence/dataInterfaces.go

```diff
@@ -1449,30 +1450,40 @@ type (
 		persistenceblobs.ImmutableClusterMetadata
 	}
 
-	// GetActiveClusterMembersRequest is the response to GetActiveClusterMembers
-	GetActiveClusterMembersRequest struct {
+	// GetClusterMembersRequest is the response to GetClusterMembers
+	GetClusterMembersRequest struct {
 		LastHeartbeatWithin time.Duration
+		RPCAddressEquals    net.IP
+		HostIDEquals        uuid.UUID
+		RoleEquals          ServiceType
+		PageSize			int
+		NextPageToken		[]byte
```

Comment:
Maybe remove for now.  You can add it back when you implement pagination.

Reply:
Whoops this wasnt supposed to make it in :)